cmd/neofs-cli/modules/object.go

```diff
@@ -185,11 +189,16 @@ func putObject(cmd *cobra.Command, _ []string) error {
 	if err != nil {
 		return err
 	}
+	btok, err := getBearerToken(cmd, "bearer")
```

Comment:
add a line break before

Reply:
I would like to ignore those whitespace linter warnings because I tried to keep changes in same style of `object.go` file. Want to keep my contribution small.